CHANGELOG.md

```diff
@@ -1,3 +1,29 @@
+## 3.1.0 (June 17, 2021)
+
+IMPROVEMENTS:
+
+-   `datadog_logs_index`: Adding missing retention_days and daily_limit parameters. ([#1083](https://github.com/DataDog/terraform-provider-datadog/pull/1083)) Thanks [@DimitryVlasenko](https://github.com/DimitryVlasenko)
+-   `datadog_dashboard`: Change Dashboard WidgetCustomLink properties. ([#1062](https://github.com/DataDog/terraform-provider-datadog/pull/1062))
```

Comment:
Can this be made a bit clearer somehow ?

Reply:
Updated the title, thanks!